lib/elixir/lib/gen_server.ex

```diff
@@ -81,6 +81,13 @@ defmodule GenServer do
   A `cast/2` message must be handled by `c:handle_cast/2`. `GenServer`
   supports 8 callbacks, but only `c:init/1` is required.
 
+  > #### `use GenServer` {: .info}
```

Comment:
Should we mention that we define also a default `handle_info/2` that logs stuff?

Reply:
That stuff is "deprecated" so I thought we should skip it. The goal is to not have it there but we only do because some people may call `super`.